prow/plugins.yaml

```diff
@@ -27,6 +27,36 @@ label:
   # responsible for maintaining the milestones:
   # curl -H "Authorization: token <token>" https://api.github.com/orgs/<org-name>/teams
   milestone_maintainers_id: 2460384
+  slacksigs:
```

Comment:
Maybe `slackchannels` instead? More importantly, what does "label" even mean here? Please document the yaml.